### ISSUE TITLE
SQL Operations using MySQL or SQLite3

### ISSUE THREAD

**Issue by krithikvaidya** [WebClub-NITK/Hacktoberfest-2k20#89]
### Description
Using MySQL or SQLite3, create a database named 'smartphones'.
Within this database, create a single table 'smartphone_details'. In this table, create the
following fields with the appropriate types and constraints:

- smartphone_id: the primary key
- model_name: the name of the smartphone's model. max length of 50 characters
- company_name: the company that created the smartphone. max length of 30 characters
- price: the price of the smartphone, in the format <rupees>.<paise> (eg: 10999.00)
- os: the base operating system of the phone. (eg: Android, iOS)
- release_date: the release date of the smartphone.

Then, insert the following tuples of data into the 'smartphone_details' table:

(1, 'Pixel 3', 'Google', 57000.00, 'Android', '2018-10-09'),
(2, 'iPhone XR', 'Apple', 49999.00, 'iOS', '2018-09-12'),
(3, 'iPhone 11 Pro', 'Apple', 99900.00, 'iOS', '2019-09-10'),
(4, 'Galaxy Note 10 Plus', 'Samsung', 85000.00, 'Android', '2019-08-07'),
(5, 'iPhone XS Max', 'Apple', 94999.00, 'iOS', '2018-09-12'),
(6, 'Galaxy A50s ', 'Samsung', 19949.00, 'Android', '2019-08-22'),
(7, 'Galaxy S9', 'Samsung', 44990.00, 'Android', '2018-03-16'),
(8, 'ROG Phone II', 'Asus', 37999.00, 'Android', '2019-09-23'),
(9, 'Mi A3', 'Xiaomi', 12999.00, 'Android', '2019-08-21'),
(10, 'Galaxy A10', 'Samsung', 7990.00, 'Android', '2019-03-02')

Now, perform the following queries on your database (Print values in all columns for each tuple):

1. Display all smartphones with price less than Rs. 60,000
2. List all smartphones released between 1st March, 2018 to 31st December, 2018.
3. Find the 2 cheapest smartphones released by Samsung after January 1, 2019 (less expensive one should be displayed first).

### Details
- Technical Specifications: The task should be done either in MySQL or SQLite. You can submit either a .sql script or a .txt file containing the commands to perform the task
- Type of issue: Multiple
- Time Limit: 1 Day

### Issue requirements / progress
- [x] MySQL
- [x] SQLite3

### Resources
[MySQL Youtube Tutorial](https://www.youtube.com/watch?v=yPu6qV5byu4)
[SQLite3 Youtube Tutorial](https://www.youtube.com/watch?v=QjICgmk31js)

### Directory Structure
Within this repository's 'systems' folder, create a folder called 'SQL_Operations' (if it doesn't exist yet). Within this folder, place the file in which you have written the script for the task. The file name should be either MySQL(.txt/.sql) or SQLite3 (.txt/.sql)

#### Note
Please claim the issue first by commenting here before starting to work on it.
While submitting a PR, request a review from @krithikvaidya
Doubts and clarifications are welcome.

**Comment by bhawesh96** [comment]
@krithikvaidya I'd like to work on this. Can you please assign this to me?

**Comment by krithikvaidya** [comment]
@bhawesh96 sure, go ahead :). Mention whether you'll be doing it in MySQL or SQLite3 though

**Comment by PriyanshTri** [comment]
can I perfrom this in Mysql?

**Comment by bhawesh96** [comment]
@PriyanshTri , I've already completed this in MySQL. Can you look for some other issue?
I'm about to submit a PR.

**Comment by krithikvaidya** [comment]
@PriyanshTri you can try it with SQLite if you wish

**Comment by henrykorir** [comment]
Assign this to me. I will do it in MySQL

**Comment by priyalgupta09** [comment]
please assign this to me i can do this in sqlite3

**Comment by krithikvaidya** [comment]
Cool, go ahead @priyalgupta09

**Comment by krithikvaidya** [comment]
> Assign this to me. I will do it in MySQL

@henrykorir sorry, it's already been done in MySQL

**Comment by adhamale2810** [comment]
Please assign it to me for sqlite3
@krithikvaidya

**Comment by adhamale2810** [comment]
Please assign it to me i have already done such database in sqlite3.
should i upload both .txt and .sqlite files or only one of them.
@krithikvaidya

**Comment by adhamale2810** [comment]
@krithikvaidya Bro i am done with the sqlite3 version on the database.
Please assign it to me so that i can make a PR

**Comment by adhamale2810** [comment]
@krithikvaidya please reply, so i can make a pr if you assign.

**Comment by adhamale2810** [comment]
@krithikvaidya please reply, whether you are assigning it to me or letting her make that pr with sqlite3

**Comment by krithikvaidya** [comment]
@adhamale2810 please be patient and give the issue creator time to respond.

Since the person who claimed it before didn't complete it within the time limit, you can go ahead.

**Comment by adhamale2810** [comment]
Sorry brother i wasn't knowing that there is some time limit.This is my first time participating in HacktoberFest

**Comment by krithikvaidya** [comment]
@adhamale2810 no problem, the purpose of Hacktoberfest is to get accustomed to contributing to open source projects :)
